keras_cv/layers/regularization/dropblock_2d.py

```diff
@@ -145,20 +153,29 @@ def __init__(
         seed=None,
         **kwargs,
     ):
-        super().__init__(seed=seed, **kwargs)
-        if not 0.0 <= rate <= 1.0:
+        # To-do: remove this once th elayer is ported to keras 3
+        # https://github.com/keras-team/keras-cv/issues/2136
+        if detect_if_tensorflow_uses_keras_3():
```

Comment:
If we want to follow the KerasNLP approach, we could just save the result of `detect_if_tensorflow_uses_keras_3` as a global in the config file, and expose a fn to check here.

`if backend.config.keras_3(): raise ValueError`

Reply:
done